internal/sdk/vendormanagement.go

```diff
@@ -412,12 +412,12 @@ func (s *VendorManagement) CreateFoundDocumentMultipart(ctx context.Context, req
 
 }
 
-// CreateFoundDocumentJSON - Upload Found Documents
+// CreateFoundDocumentMultipart - Upload Found Documents
 // Upload a newly discovered document (e.g. Order Form, Quote) to Lumos for review
-func (s *VendorManagement) CreateFoundDocumentJSON(ctx context.Context, request shared.FoundDocumentsUpload, opts ...operations.Option) (*operations.CreateFoundDocumentJSONResponse, error) {
+func (s *VendorManagement) CreateFoundDocumentMultipart(ctx context.Context, request operations.CreateFoundDocumentMultipartRequestBody, opts ...operations.Option) (*operations.CreateFoundDocumentMultipartResponse, error) {
```

Comment:
where is this multipart thing coming from? I vaguely recall removing it from the API?

Reply:
This part is black magic to me to be honest. Part of the downside of relying on a generated provider -- it's less explainable to see what this change actually means